gapii/cc/spy_base.h

```diff
@@ -33,6 +33,10 @@
 #include "core/cc/coder/memory.h"
 #include "core/cc/coder/atom.h"
 
+#if (TARGET_OS == GAPID_OS_LINUX) || (TARGET_OS == GAPID_OS_ANDROID)
+#include <core/memory_tracker/cc/memory_tracker.h>
```

Comment:
If you were to add the `USE_MEMORY_TRACKER` define, I'd recommend it was defined in `memory_tracker.h`